cmd/flags/signature_aggregator.go

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package flags
+
+import (
+	"fmt"
+
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/ava-labs/avalanchego/utils/logging"
+	"github.com/spf13/cobra"
+)
+
+const (
+	aggregatorLogLevelFlag    = "aggregator-log-level"
+	aggregatorLogToStdoutFlag = "aggregator-log-to-stdout"
+)
+
+var SigAggFlags SignatureAggregatorFlags
+
+type SignatureAggregatorFlags struct {
+	AggregatorLogLevel    string
+	AggregatorLogToStdout bool
+}
+
+func validateSignatureAggregatorFlags() error {
+	if _, err := logging.ToLevel(SigAggFlags.AggregatorLogLevel); err != nil {
+		return fmt.Errorf(
+			"invalid log level: %q. Available values: %s, %s, %s, %s, %s, %s, %s, %s",
+			SigAggFlags.AggregatorLogLevel,
+			logging.Info.LowerString(),
+			logging.Warn.LowerString(),
+			logging.Error.LowerString(),
+			logging.Off.LowerString(),
+			logging.Fatal.LowerString(),
+			logging.Debug.LowerString(),
+			logging.Trace.LowerString(),
+			logging.Verbo.LowerString(),
+		)
+	}
+	return nil
+}
+
+func AddSignatureAggregatorFlagsToCmd(cmd *cobra.Command) {
+	cmd.Flags().StringVar(&SigAggFlags.AggregatorLogLevel, aggregatorLogLevelFlag, constants.DefaultAggregatorLogLevel, "log level to use with signature aggregator")
```

Comment:
Where are we setting the DefaultAggregatorLogLevel  values for the flags currently?

Reply:
constants.DefaultAggregatorLogLevel is defined in https://github.com/ava-labs/avalanche-cli/blob/main/pkg/constants/constants.go#L346

Reply:
probably we need to move default flag values into flags pkg?